### DISCUSSION TITLE
会配置鸿蒙Next吗？

### DISCUSSION THREAD

**Discussion by walkerandmay** [xiaojieonly/Ehviewer_CN_SXJ#1447] [Q&A]
华为手机，升级Next的最大阻力就是舍不得Ehviewer啊啊啊啊

**Comment by xiaojieonly** [comment]
不会，我没钱买华为手机

**Comment by xiaojieonly** [comment]
我去看了下，华为的next是闭源系统，就像是苹果一样，所有app都只能从应用市场安装
看到这里我就没看了，搞出来注定过不了审的。
个人建议有需求的乘早换别的手机

**Comment by GiveMeOneLastKiss** [comment]
目前不太可能，只能看后期华为会不会开放第三方软件安装了
